src/argilla/training/base.py

```diff
@@ -234,6 +236,12 @@ def __init__(
             )
 
         self._logger.info(self)
+        self._track_trainer_usage(framework=framework, task=self._rg_dataset_type._RECORD_TYPE.__name__)
+
+    def _track_trainer_usage(self, framework: str, task: str):
+        if not self._CLIENT:
+            return
+        self._CLIENT.track_data(action="ArgillaTrainerUsage", data={"framework": framework, "task": task})
```

Comment:
We should update the telemetry [section in the docs](https://github.com/argilla-io/argilla/pull/3346) to include this new tracking.